<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Automata" ignored="0" name="Ups Login Test" tests="2" failures="1" timestamp="30 Nov 2020 04:32:00 GMT" time="23.675" errors="0">
  <testcase name="loginTest" time="1.828" classname="com.ups.testRunner.UpsTestRunner">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.ups.testRunner.UpsTestRunner.loginTest(UpsTestRunner.java:33)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="noTest" time="0.001" classname="com.ups.testRunner.UpsTestRunner"/>
</testsuite> <!-- Ups Login Test -->
